adhocracy4/organisations/predicates.py

```diff
@@ -26,9 +26,17 @@ def is_org_member(user, subject):
 
 
 @rules.predicate
-def is_org_group_member(user, organisation):
-    if organisation:
-        if hasattr(organisation, 'groups'):
+def is_org_group_member(user, subject):
+    if subject:
+        if hasattr(subject, 'has_initiator') and hasattr(subject, 'groups'):
+            organisation = subject
+            org_groups = organisation.groups.all()
+            user_groups = user.groups.all()
+            group = org_groups & user_groups
```

Comment:
Nitpick :) : What do you think about taking these out of the if statement and combining them with lines 40-42?

Reply:
Haha, late brain only thought about the return outside of the ifs. But that can also go in the else! Yes, good point!

Reply:
And I didn't even need the else!

Reply:
Perfect!